lib/wallaby/browser.ex

```diff
@@ -857,6 +857,16 @@ defmodule Wallaby.Browser do
     end
   end
 
+  defp do_get(query, elements) do
+    getNumber = query.conditions[:get]
+    case {getNumber, length(elements)} do
+        {nil,_}     -> {:ok, elements}
+        {n, count} when n > 0 and n <= count
+                    -> {:ok, [Enum.at(elements, n-1)]}
+        {_, _}      -> {:error, {:get_number, length(elements)}}
```

Comment:
smaller style pet peeve(not too important and our fault for not having credo setup) is that I like all case statements to be one line or all of them to be multiline - not a biggie though :)